### PR TITLE
[astro] Update README.md

### LINKED ISSUE #15005: [astro] Documentation missing: sun:phase:name NOON
The current documentation is missing that there is a sun phase name "NOON":

> 2023-05-18 13:41:47.307 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'SunPhase' changed from DAYLIGHT to NOON
> 2023-05-18 13:42:47.232 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'SunPhase' changed from NOON to DAYLIGHT

It's short but it upset my rule controlling the hall lights in any state other than "DAYLIGHT"...

It would be great if this could be added to the documentation.

Many thanks in advance!

### PR DESCRIPTION
**Pull request by boehan**
Fixes #15005 

~`MORNING_NIGHT` and `EVENING_NIGHT` seem to be missing as well acc. to https://github.com/openhab/openhab-addons/blob/4d811691e91f348217a1855abc7c22f1a6669ac7/bundles/org.openhab.binding.astro/src/main/java/org/openhab/binding/astro/internal/model/SunPhaseName.java#L23C1-L37~ removed again

Signed-off-by: Hans Böhm <h.boehm@gmx.at>

### CODE REVIEW

**Review by lsiepel — APPROVED** [review]
LGTM

**Review by clinique — APPROVED** [review]

File: bundles/org.openhab.binding.astro/README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ This is done by setting `useMeteorologicalSeason` to true in the advanced settin
       - `totalElevation, partialElevation, ringElevation` (Number:Angle)
   - **group** `phase`
     - **channel**
-      - `name` (String), values: `SUN_RISE, ASTRO_DAWN, NAUTIC_DAWN, CIVIL_DAWN, CIVIL_DUSK, NAUTIC_DUSK, ASTRO_DUSK, SUN_SET, DAYLIGHT, NIGHT`
+      - `name` (String), values: `SUN_RISE, ASTRO_DAWN, NAUTIC_DAWN, CIVIL_DAWN, CIVIL_DUSK, NAUTIC_DUSK, ASTRO_DUSK, SUN_SET, DAYLIGHT, NOON, NIGHT, MORNING_NIGHT, EVENING_NIGHT`
```
Comment (lolodomo) [inline bundles/org.openhab.binding.astro/README.md:60]: I do not find MORNING_NIGHT and EVENING_NIGHT in channel type description:
https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.astro/src/main/resources/OH-INF/thing/channels.xml#L283

Comment (clinique) [reply bundles/org.openhab.binding.astro/README.md:60]: They are present in the java enum :https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.astro/src/main/java/org/openhab/binding/astro/internal/model/SunPhaseName.java

Comment (boehan) [reply bundles/org.openhab.binding.astro/README.md:60]: I'm not sure, if they are used actually and if yes, I've no idea when. Tried to trigger it manually (by changing locations), but I couldn't: `DAYLIGHT` is followed by `SUN_SET` and the different `*_DUSK` then before `NIGHT` but apparently no `EVENING_NIGHT` between (or the interval is so short, that my manual tries didn't match it).

So we have two options here:
1. also add it to the channel type descriptions, or
2. remove them in the README again

WDYT?

Comment (clinique) [reply bundles/org.openhab.binding.astro/README.md:60]: If they are not fired, I would suggest to remove them from documentation.

Comment (boehan) [reply bundles/org.openhab.binding.astro/README.md]: Done, even though I'm not sure if they could eventually trigger in some other configuration...

**Comment by boehan** [comment]
DCO check does not pass as I made the change directly in Github (but signoff is included in commit message)

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Pull request merged**